xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -4563,6 +4622,7 @@ absl::Status IrEmitterUnnested::EmitHloInstruction(
     case HloOpcode::kGetTupleElement:
     case HloOpcode::kParameter:
     case HloOpcode::kTuple:
+    case HloOpcode::kCopyDone:
```

Comment:
Does this mean that the copy-done is a no-op? In Hlo, we expect that this op would block until the copy is completed. Is that how this works?

Reply:
Treating copy-done as a no-op seems working. I'll double check with my coworker.

Reply:
I got an nsys report showing that the memcpy H2D/D2H that I generated are in the same stream (Stream #118) of other compute operations like wrapped_add_1. So treating copy-done as a no-op works now. The future PR will use a separate stream for memcpy and the blocking or events will be absolutely necessary when handling copy-done. So far we can leave copy-done as is for the current PR.
<img width="1314" alt="nsys-offloading" src="https://github.com/openxla/xla/assets/90360117/5bbf092e-70a8-493d-8ef6-125fd29ca476">